tests/test_venue_request.py

```diff
@@ -2037,3 +2042,20 @@ def test_supplementary_material_revision(self, client, test_client, selenium, re
         assert all(x not in revision_invitations[0].reply['content'] for x in ['title','authors', 'authorids','abstract','keywords', 'TL;DR'])
         assert revision_invitations[0].duedate
         assert revision_invitations[0].expdate
+
+        #make sure homepage webfield was not overwritten after doing get_conference()
```

Comment:
this is testing the decision tabs are still there even after using the request form

Reply:
excelent!